src/style/utils/add-form-spacing.spec.ts

```diff
@@ -0,0 +1,51 @@
+import addFormSpacing from "./add-form-spacing";
+
+describe("addFormSpacing", () => {
+  it("does not set the mb and mt properties if passed props is empty object and no fieldSpacing value", () => {
+    expect(addFormSpacing({})).toEqual({});
+  });
+
+  it("sets the mb and mt properties if passed props is empty object", () => {
+    expect(addFormSpacing({}, "5px")).toEqual({ mb: "5px", mt: 0 });
+  });
+
+  it.each(["mx", "marginX", "ml", "marginLeft", "mr", "marginRight"])(
+    "sets the mb and mt properties if %s passed and no custom margin bottom exists in props",
+    (marginProp) => {
+      expect(addFormSpacing({ [marginProp]: "10px" }, "5px")).toEqual({
+        [marginProp]: "10px",
+        mb: "5px",
+        mt: 0,
+      });
+    }
+  );
+
+  it.each(["mt", "marginTop"])(
+    "only sets the mb property if %s passed and no custom margin bottom exists in props",
```

Comment:
Not sure if this description is accurate - in the test aren't you setting the margin bottom?

Reply:
I'll tweak this slightly as it's unclear, it should say that it sets the mb to the formSpacing value if no value exists in the prop object